akka-remote/src/main/scala/akka/remote/artery/Association.scala

```diff
@@ -409,6 +408,8 @@ private[remote] class Association(
   }
 
   private def runOutboundStreams(): Unit = {
+    if (transport.isShutdown) throw ShuttingDown
```

Comment:
I think this should be moved to each run*Stream method, e.g. runOutboundControlStream, becuase those are used independently for restarts